outlines/ui-ux-outline.md

```diff
@@ -0,0 +1,28 @@
+# UI / UX
+
+1. Building reusable components to encapsulate UX patterns
+  1. Pure vs impure in general
+  2. Meteor's global singletons
+  3. A component harness / styleguide
+2. Event handling
+  1. Throttling
+3. Responsive design
+  1. Very basic ideas using media queries
+  2. Reference mobile chapter, talk about Cordova wrapper
+4. Accessiblity
+5. Internationalization - using `tap:i18n`
```

Comment:
This section should include a list of all of the things that need to be i18ized - error messages is one non-obvious example, there might be more.